tools/tck-api/src/main/scala/org/opencypher/tools/tck/api/CypherTCK.scala

```diff
@@ -129,18 +131,25 @@ object CypherTCK {
           // filters out scenarios with TCKTags.IGNORE
           val included = pickles.filterNot(tagNames(_) contains TCKTags.IGNORE)
 
-          val includedGroupedByName = included.groupBy(_.getName)
-          val includedGroupedAndSorted = includedGroupedByName.mapValues(_.sortBy(_.getLocation.getLine))
+          val includedWithExtractedName = included.map(p => {
+            p.getName match {
+              case pickleNamePattern1(pickleName, _, null) => (p, pickleName, None)
+              case pickleNamePattern1(pickleName, _, exampleName) => (p, pickleName, Some(exampleName))
```

Comment:
Is it worth creating a case class here, ._1, ._2, ._3 are not very descriptive?

Reply:
I will address this.